src/ad-units.js

```diff
@@ -37,50 +35,38 @@ var AdUnits = {
     var closedClass = 'closed';
     var toggleButton = adElement.nextElementSibling;
 
-    if (utils.hasClass(toggleButton, 'open')) {
-      utils.removeClass(toggleButton, openedClass);
-      utils.removeClass(adElement.parentElement, openedClass);
-      utils.addClass(toggleButton, closedClass);
-      utils.addClass(adElement.parentElement, closedClass);
+    if (toggleButton.classList.contains('open')) {
```

Comment:
contains can be problematic in ie. MDN says it ie 5+ supports it (https://developer.mozilla.org/en-US/docs/Web/API/Node/contains), but I think it only applies to dom elements https://connect.microsoft.com/IE/feedback/details/780874/node-contains-is-incorrect